### PR TITLE
Assembly: define joint double click on 3dview

### PR DESCRIPTION
**Pull request by PaddleStroke**
Enable to edit joint by double clicking them in the 3d view.
Fix https://github.com/FreeCAD/FreeCAD/issues/14701

### CODE REVIEW

File: src/Mod/Assembly/Gui/ViewProviderAssembly.cpp [hunk 1]
```diff
@@ -38,6 +38,8 @@
 #include <Inventor/sensors/SoSensor.h>
 #endif
 
+#include <chrono>
```
Comment (PaddleStroke) [inline src/Mod/Assembly/Gui/ViewProviderAssembly.cpp:41]: What to do about this Link warning? @chennes 

> <chrono> is an unapproved C++11 header.  [build/c++11] [5]

Comment (chennes) [reply src/Mod/Assembly/Gui/ViewProviderAssembly.cpp:41]: That warning doesn't apply to our project (it's from Google's internal guidelines). We should adjust our linter settings to turn it off.

Comment (PaddleStroke) [reply src/Mod/Assembly/Gui/ViewProviderAssembly.cpp:41]: Thanks

File: src/Mod/Assembly/Gui/ViewProviderAssembly.cpp [hunk 2]
```diff
@@ -459,6 +475,36 @@ bool ViewProviderAssembly::mouseButtonPressed(int Button,
     return false;
 }
 
+void ViewProviderAssembly::doubleClickedIn3dView()
+{
+    // Double clicking on a joint should start its edition.
```
Comment (chennes) [inline src/Mod/Assembly/Gui/ViewProviderAssembly.cpp:480]: "Edition" == "addition"?

**Review by chennes — APPROVED** [review]
Compiles without warning on MSVC, no linter issues -- just a question about the documentation comment. This is adding a joint, right? Not editing one?

**Comment by PaddleStroke** [comment]
No it's editing a joint not adding one.
When you double click on an existing joint in the 3d view, it does now the
same as double clicking it in the tree : edit the joint.

File: src/Mod/Assembly/Gui/ViewProviderAssembly.cpp (hunk 2 above)
Comment (chennes) [reply src/Mod/Assembly/Gui/ViewProviderAssembly.cpp:480]: Not actually resolved: "edition" is not the word you are looking for here. Try just "edit", or reword to "start editing it" or "enters edit mode" or something.

Comment (PaddleStroke) [reply src/Mod/Assembly/Gui/ViewProviderAssembly.cpp:480]: Ah ok

**Comment by PaddleStroke** [comment]
String updated

**Pull request merged**